Automattic/jetpack
# Forms/add response defaults

This PR adds some default data/structure to the responses inbox API route.

## Proposed changes:
Providing these default data sets avoids getting PHP warnings on the logs and makes it simpler on the code

### Other information:



## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Tweak the code to return empty sets:
`projects/packages/forms/src/class-wpcom-rest-api-v2-endpoint-forms.php` line 202, replace `$query[ $current_query ]->posts` for an empty array
```
			},
			[] // $query[ $current_query ]->posts
		);
```

Reload the responses inbox page. Keep an eye on logs with `jetpack docker tail`. See that no warnings are raised.
